tests/testthat/test-exposure_metrics.R

```diff
@@ -169,3 +169,104 @@ test_that("handles IOV correctly", {
   expect_false(result2a == result2b) # iov is different after day 3
 })
 
+test_that("calc_time_to_target: correct time to target calculated", {
```

Comment:
it would be good to add a test for when a supported target is used but the encounter never attains target.